config.yaml

```diff
@@ -63,6 +67,7 @@ stale_critical:
         This issue was labeled "{warning_label}" {warning_days} ago and has not received any updates so it is being deprioritized. If this ticket is actually Critical, please raise the priority and ask a committer to assign you the issue or revive the public discussion.
 
 stale_major:
+    ticket_limit: 15
```

Comment:
Every rule performs two runs: one run to mark tickets stale and another run to perform an action on tickets marked stale. ticket_limit applies to each of the runs, so that overall 100 tickets are touched at most.